ci/test/06_script_a.sh

```diff
@@ -6,6 +6,18 @@
 
 export LC_ALL=C.UTF-8
 
+lcd=cd
+if [[ "$OSTYPE" == "darwin"* ]]; then
+  lcd=type cd
+fi
+
+safe_cd()
```

Comment:
at least add a comment for this workaround here: why this is needed, and when it can be removed again

Reply:
Ok, thanks.
I'll add.